packages/grid/_modules_/grid/components/GridViewport.tsx

```diff
@@ -78,6 +78,7 @@ export const GridViewport: ViewportType = React.forwardRef<HTMLDivElement, {}>(
             cellTabIndex={cellTabIndex}
             isSelected={selectionState[id] !== undefined}
             editRowState={editRowsState[id]}
+            classes={options.classes?.cell}
```

Comment:
This breaks the terminology convention. `classes` is an object, here it seems to be a string. What's wrong, with `className`?

Reply:
Nothing I just thought carrying the name. I will rename it